components/automate-ui/src/app/page-components/services-sidebar/services-sidebar.component.scss

```diff
@@ -330,3 +331,31 @@ chef-alert {
   font-size: 14px;
   word-break: break-all;
 }
+
+.bulk-action-bar {
+  display: flex;
+  align-items: center;
+  padding: 0 35px;
+  // temp - check with UX
+  position: sticky;
+  top: 0;
+  z-index: 10;
+  background: $chef-white;
```

Comment:
@apriofrost - I made the bulk bar sticky so that on long service amounts, a user could scroll and see how many elements are selected/select all/delete services without having to scroll back up the page.  It seemed useful in my testing because I had to keep scrolling back up to see what I was doing.  Leave it in?  Pull it?  Change it?